setuptools/config/pyprojecttoml.py

```diff
@@ -28,10 +29,13 @@
 
 
 def load_file(filepath: _Path) -> dict:
-    from setuptools.extern import tomli  # type: ignore
+    if sys.version_info >= (3, 11):
+        import tomllib
+    else:
```

Comment:
Should we add 

```suggestion
    else:  # pragma: no cover
```

Just to get rid of the annoying coverage warning?

Reply:
If you want me to, sure.

This (among other reasons) is why I simply don't care about or use coverage. I have no opinions on it other than "if the maintainer asks me to do it then I can oblige".

Reply:
Probably updating the version of python used in coverage runs is going to require repetitively reworking coverage pragma ignores, but as a coverage using project you're no doubt used to this kind of churn.

Reply:
Pushed, fingers crossed.